streaming/java/streaming-runtime/src/test/java/org/ray/streaming/runtime/TestHelper.java

```diff
@@ -0,0 +1,18 @@
+package org.ray.streaming.runtime;
+
+import org.ray.runtime.RayNativeRuntime;
+import org.ray.runtime.util.JniUtils;
+
+public class TestHelper {
+
+  public static void loadNativeLibraries() {
+    // load core_worker_library_java before load streaming_java
+    try {
+      Class.forName(RayNativeRuntime.class.getName());
```

Comment:
```
    try {
      // Explicitly load `RayNativeRuntime`, to make sure `core_worker_library_java` is loaded before `streaming_java`.
      Class.forName(RayNativeRuntime.class.getName());
```
This would be clearer.

Reply:
fixed